docs/source/options/dash_options.rst

```diff
@@ -95,3 +95,8 @@ DASH options
 
     If enabled, allow adaptive switching between different codecs, if they have 
     the same language, media type (audio, video etc) and container type.
+
+--is_low_latency_dash
```

Comment:
Same comment I had on the streamer side: let's not use "is".  :-)

Reply:
Good call! I just updated the naming here. I also made a PR to address the name change in the rest of the files: https://github.com/CaitlinOCallaghan/shaka-packager/pull/7